src/main/java/org/sonatype/gradle/plugins/scan/common/DependenciesFinder.java

```diff
@@ -83,8 +84,13 @@
       Module module = buildModule(project);
 
       findResolvedArtifacts(project, allConfigurations).forEach(resolvedArtifact -> {
-        Artifact artifact = new Artifact().setId(resolvedArtifact.getId().getComponentIdentifier().getDisplayName())
-            .setPathname(resolvedArtifact.getFile()).setMonitored(true);
+        ModuleVersionIdentifier artifactId = resolvedArtifact.getModuleVersion().getId();
+
+        Artifact artifact = new Artifact()
+            .setId(artifactId.getGroup() + ":" + artifactId.getName() + ":" + artifactId.getVersion())
```

Comment:
Does the `ModuleVersionIdentifier` has any info on `classifier` and `extension` ?

Reply:
> Does the `ModuleVersionIdentifier` has any info on `classifier` and `extension` ?

Nope!
Only group, name and version: https://docs.gradle.org/current/javadoc/org/gradle/api/artifacts/ModuleVersionIdentifier.html